### PR TITLE
educate about 'chaining' debug()

### PR DESCRIPTION
**Pull request by ionas**
Ref: https://github.com/cakephp/cakephp/pull/8534#issuecomment-201585305

### CODE REVIEW

File: en/development/debugging.rst [hunk 1]
```diff
@@ -24,6 +24,13 @@ default.
 Output from this function is only shown if the core ``$debug`` variable
 has been set to ``true``.
 
+.. versionadded:: 3.2.6
```
Comment (markstory) [inline en/development/debugging.rst]: I think this won't be in until 3.3.0.

File: en/development/debugging.rst [hunk 2]
```diff
@@ -24,6 +24,13 @@ default.
...
+    Calling this method will return passed ``$var``, so that you can, for instance,
+    place it in return statements, for example: ``return debug($data)`` will return
```
Comment (lorenzo) [inline en/development/debugging.rst]: Use a code block is better for the eyes than reading it in-line :)

**Comment by ionas** [comment]
Can you change the target to 3.3.0?

**Comment by lorenzo** [comment]
No, you need to close this one

**Pull request closed without merging**